Rakefile

```diff
@@ -146,8 +146,18 @@ end
 
 namespace :release do
   namespace :version do
+    desc "Validate version for release"
+    task :validate do
+      if version != BASE_VERSION
+        message = "You must NOT specify VERSION= for 'rake release'. "
+        message += "You must run 'rake dev:version:bump NEW_VERSION=#{version}' "
+        message += "before 'rake release'."
+        raise message
+      end
+    end
+
     desc "Update versions for a new release"
-    task :update do
+    task update: :validate do
```

Comment:
Could you add the same thing to `release:tag` as well?

Reply:
fix: [3838f2e](https://github.com/groonga/groonga/pull/2504/commits/3838f2e50256d87fa5a2c16980eb2e3651833b25) Sure!